### ISSUE TITLE
Add 'Fees on top' to contribution flow

### ISSUE THREAD

**Issue by BenJam** [opencollective/opencollective#4141]
## User story 

> I want to donate a specific amount to a project to pay for a proposed or anticipated cost (let's say $50 for domain hosting), I'm happy to pay for host fees but I need the project to receive the amoutn of money to cover that cost. 

## Best solution for this problem
Let me add the fees (host, payment processor) 'on top' of my donation, rather than deduct them. 

## MVP
A simple hack to the existing interface that adds a checkbox below the donation step that says 'add x to cover y and z fees:

![ontopfees](https://user-images.githubusercontent.com/158833/113322682-e6712f00-930c-11eb-9448-e4318c2291a3.png)

## Metrics 
Increase in average contribution, positive feedback.

**Comment by BenJam** [comment]
I'm almost sure that @alanna has pitched this before but I UTFS'd and could not find the issue. Please do link and close if needed. Thank you.

**Comment by alanna** [comment]
I support this conceptually, and it aligns with how some other crowdfunding platforms work. It's also an appealing proposition for major funders, who often are happy to pay overheads and want a certain amount to go to the funded initiative. Right now we do this manually by adding X% on top of the invoice in the first place so when we add the funds the right amount goes to the Collective after fees, but it would make sense to have a self-serve version of it too that does the math for you.

However, the reason we haven't pursued this and I  think why I didn't raise an issue after we talked about it is, unlike other platforms we have added complexity that would probably make this super confusing. People are already asked to give a platform tip on top of their contribution. Adding yet another box for "fiscal host tip" is too much. People *do not* get the whole fiscal hosting concept, even hosted Collectives don't get it much less their contributors. We can't explain it well in the context of a contribution flow we're trying to make as lightweight as possible. 

The logical next idea would be to just as for one tip covering all fees, and then split that between the host and the platform in the background, but that's just recreating our old funding model which have spent the last couple years moving on from. I really don't think that's the answer, unless we change platform tips to an optional "cover fees?" ask, without reference to platforms and hosts, and then applied the normal OC 15% of host fees on it. But that would mean asking hosts to be ok with an optional revenue model and most hosts won't be up for that because they have very real costs that aren't flexible and have to charge their fees.

**Comment by BenJam** [comment]
> People are already asked to give a platform tip on top of their contribution

Noted. It's likely that my use case is quite small as i was trying to give a certain amount.

**Comment by utunga** [comment]
Yeah this would be a great feature I agree.. sometimes the contributor is contributing for a specific reason and so they're trying to ensure a certain amount arrives in the actual organisation account at the end of the day (this is what happened to us the other day)
